scripts/ci

```diff
@@ -33,13 +34,17 @@ function build() {
         export PATH="/usr/local/opt/llvm/bin:/usr/local/bin:${PATH}"
         export PKG_CONFIG_PATH="/usr/local/opt/openssl/lib/pkgconfig"
         export LIBRARY_PATH="/usr/local/opt/openssl/lib"
-        export LDFLAGS="-L/usr/local/opt/llvm/lib -L-L/usr/local/lib"
-        export CPPFLAGS="-I/usr/local/opt/llvm/include -I/usr/local/include"
+        export LDFLAGS="-L/usr/local/lib"
+        export CPPFLAGS="-I/usr/local/include"
 
+        cxx_flags="${cxx_flags} -I/usr/local/opt/openssl/include"
         cxx_flags="${cxx_flags} -Wno-deprecated-declarations"
     fi
 
     cmake_flags="-DCMAKE_BUILD_TYPE=${build_type}"
+    if ! [ "1" == "${full}" ] ; then
```

Comment:
`"1" == "${full}"` let's switch it to `"${full}" == "1"` as it reads more naturally -> "if full is true/1" instead of "if 1/true is full"

Reply:
Changed the order in conditions